libs/interpreter-lib/src/std-extension.spec.ts

```diff
@@ -5,25 +5,61 @@
 import { strict as assert } from 'assert';
 
 import { getRegisteredBlockExecutors } from '@jvalue/jayvee-execution';
-import { getRegisteredBlockMetaInformation } from '@jvalue/jayvee-language-server';
+import {
+  BlockMetaInformation,
+  createJayveeServices,
+  initializeWorkspace,
+  isBuiltinBlocktypeDefinition,
+  isJayveeModel,
+} from '@jvalue/jayvee-language-server';
+import { NodeFileSystem } from 'langium/node';
 
 import { useStdExtension } from './interpreter';
 
+async function getAllBuiltinBlocktypes(): Promise<BlockMetaInformation[]> {
+  const allBuiltinBlocktypes: BlockMetaInformation[] = [];
+  const services = createJayveeServices(NodeFileSystem).Jayvee;
+  await initializeWorkspace(services);
+  const documentService = services.shared.workspace.LangiumDocuments;
+
+  documentService.all
+    .map((document) => document.parseResult.value)
+    .forEach((parsedDocument) => {
+      if (!isJayveeModel(parsedDocument)) {
+        throw new Error('Expected parsed document to be a JayveeModel');
+      }
+      parsedDocument.blocktypes.forEach((blocktypeDefinition) => {
+        if (!isBuiltinBlocktypeDefinition(blocktypeDefinition)) {
+          return;
+        }
+        if (BlockMetaInformation.canBeWrapped(blocktypeDefinition)) {
+          allBuiltinBlocktypes.push(
+            new BlockMetaInformation(blocktypeDefinition),
+          );
+        }
+      });
+    });
+  return allBuiltinBlocktypes;
+}
+
 describe('std extension', () => {
-  useStdExtension();
-  getRegisteredBlockMetaInformation().forEach((metaInf) => {
-    it(`should provide a matching block executor for block type ${metaInf.type}`, () => {
-      const matchingBlockExecutorClass = getRegisteredBlockExecutors().find(
-        (blockExecutorClass) => blockExecutorClass.type === metaInf.type,
-      );
+  it('should provide matching block executors for builtin block types', async () => {
+    useStdExtension();
+    (await getAllBuiltinBlocktypes()).forEach(
```

Comment:
Unfortunately, I wasn't able to pull that out so that each blocktype has its own test case. Issue is the asynchronous nature of loading the builtin blocktypes. Requires some further digging I guess.

The function `getAllBuildinBlocktypes` might be a good candidate to pull out and add as util functionality to the `interpreter-lib`

Reply:
Pulled out function to `model-util`.